### PR TITLE
114106375 how old master

### PR DESCRIPTION
**Pull request by Danielsmithitguy**

### CODE REVIEW

File: numbers_letters_and_variables/number_of_seconds.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+
+seconds_in_min = 60
```
Comment (jasonnoble) [inline numbers_letters_and_variables/number_of_seconds.rb]: Short variables make me :cry:

File: numbers_letters_and_variables/number_of_seconds.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+#  interger_valid = true
+#end
+#end
+
+puts "There are #{seconds_in_min} seconds in a minute"
+puts "There are #{min_in_hour} minutes in an hour"
+puts "There are #{hours_in_day} hours in a day"
+puts "There are #{days_in_week} days in a week"
+puts "That means there are:"
+puts "  #{seconds_in_min * min_in_hour} seconds in an hour,"
```
Comment (jasonnoble) [inline numbers_letters_and_variables/number_of_seconds.rb]: Pull these out into a `seconds_per_hour` variable?

File: numbers_letters_and_variables/number_of_seconds.rb [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+puts "  #{seconds_in_min * min_in_hour * hours_in_day} seconds in an day,"
+puts "  #{seconds_in_min * min_in_hour * hours_in_day * days_in_week} seconds in a week"
```
Comment (jasonnoble) [inline numbers_letters_and_variables/number_of_seconds.rb]: IMO, too much being done in interpolation.  Pull this computation out into a variable?

**Comment by anthonyfalzetti** [comment]
[Accepts #11410637]

**Pull request merged**